### DISCUSSION TITLE
用docker快速开始，没提示任何异常镜像直接关闭

### DISCUSSION THREAD

**Discussion by yicong0513** [apache/streampark#2811] [Q&A] [answered]
streampark_streampark-console_1 exited with code 0
![image](https://github.com/apache/incubator-streampark/assets/37571946/edd7d7e4-2b3a-4b8d-a889-1785bb20f6dc)
![image](https://github.com/apache/incubator-streampark/assets/37571946/a1c16287-b986-4ecb-84b0-eb6f4df37759)

**Comment by caicancai** [comment, marked as answer]
Hello, I have no problem starting streampark with docker. You can check the error log inside docker to find out the reason.
